docker-compose.yml

```diff
@@ -1,7 +1,3 @@
-# By default, this docker-compose setup uses Kafka 2.7.0. This version can
```

Comment:
all I did here was copy and paste in @fpacifici's docker-compose which uses confluent instead of bitnami kafka. We can revert this if we ever contribute back to upstream (but for now this is handy for local testing)